### PR TITLE
report(redesign): toggle dark theme in menu

### PR DESCRIPTION
**Pull request by Hoten**
#8185

I know the drop shadow is wonky but that gets fixed in one of these other prs.

### CODE REVIEW

**Comment by Hoten** [comment]
should i rename everything from `export` -> `menu`?

File: lighthouse-core/report/html/templates.html [hunk 1]
```diff
@@ -351,6 +351,7 @@
         <a href="#" class="report-icon report-icon--download" data-action="save-json">Save as JSON</a>
         <a href="#" class="report-icon report-icon--open lh-export--viewer" data-action="open-viewer">Open in Viewer</a>
         <a href="#" class="report-icon report-icon--open lh-export--gist" data-action="save-gist">Save as Gist</a>
+        <a href="#" class="report-icon report-icon--open lh-export--dark" data-action="toggle-dark">Toggle Dark Theme</a>
```
Comment (paulirish) [inline lighthouse-core/report/html/templates.html]: lets use a real icon for this. 
lightbulb seems good. 💡

Comment (Hoten) [reply lighthouse-core/report/html/templates.html]: ok but i dont love how i did it..

**Comment by paulirish** [comment]
Use the lightbulb in material icons?

>

**Comment by Hoten** [comment]
"highlight"?
https://material.io/tools/icons/?search=light&style=baseline

**Comment by paulirish** [comment]
@cjamcl i totally messed with your PR. can you give it a review?

**Comment by Hoten** [comment]
all good

**Comment by paulirish** [comment]
fyi: i checked the `filter: invert()` in FF and Saf and we good. 😎

**Pull request merged**